functions/notify_slack.py

```diff
@@ -51,6 +51,8 @@ def ecs_notification(message, region):
     for key in message['detail']['containers']:
       if 'reason' in key:
         fields.append( { "title": "reason", "value": key['reason'], "short": True } )
+        fields.append( { "title": "taskArn", "value": key['taskArn'], "short": True } )
```

Comment:
Should we take out the taskDefinitionArn if we're listing the taskArn then? Or maybe the taskArn is better all the time since we can still tell which service this is for.

Reply:
Yea, seems like taskArn is repeated if I add that here. So we could change taskDefinitionArn to just taskArn further down in this ecs_notification def.